Scripts/Editor/Core/AnimationSequencerControllerCustomEditor.cs

```diff
@@ -47,7 +46,13 @@ private void OnEnable()
             reorderableList.onReorderCallback += OnListOrderChanged;
             reorderableList.drawHeaderCallback += OnDrawerHeader;
             EditorApplication.playModeStateChanged += OnEditorPlayModeChanged;
+            
+#if UNITY_2021_1_OR_NEWER
```

Comment:
This should be UNITY_2021_2_OR_NEWER since this was still experimental in 2021.1 version as you can see here https://docs.unity3d.com/2021.1/Documentation/ScriptReference/Experimental.SceneManagement.PrefabStage.html